providers/amazon/tests/unit/amazon/aws/sensors/test_mwaa.py

```diff
@@ -0,0 +1,72 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+from __future__ import annotations
+
+from unittest import mock
+
+import pytest
+
+from airflow.exceptions import AirflowException
+from airflow.providers.amazon.aws.hooks.mwaa import MwaaHook
+from airflow.providers.amazon.aws.sensors.mwaa import MwaaDagRunSensor
+from airflow.utils.state import State
+
+SENSOR_KWARGS = {
+    "task_id": "test_mwaa_sensor",
+    "external_env_name": "test_env",
+    "external_dag_id": "test_dag",
+    "external_dag_run_id": "test_run_id",
+}
+
+
+@pytest.fixture
+def mock_invoke_rest_api():
+    with mock.patch.object(MwaaHook, "invoke_rest_api") as m:
+        yield m
+
+
+class TestMwaaDagRunSuccessSensor:
+    def test_init(self):
+        s_states = {"state1", "state2"}
+        f_states = {"state3", "state4"}
```

Comment:
minor nit: But I don't love single character variable names (or even components of a variable name with single characters). I think it's much cleaner an readable to have `success_states` and `failure_states` and it's really not thaaat much longer.

Reply:
Fair, fixed!